models/stg_sap.yml

```diff
@@ -0,0 +1,2158 @@
+        description: Number
+      - name: anz40
+        description: Number
+      - name: bet01
+        description: Wage type amount for payments
+      - name: bet02
+        description: Wage type amount for payments
+      - name: bet03
+        description: Wage type amount for payments
+      - name: bet04
+        description: Wage type amount for payments
+      - name: bet05
+        description: Wage type amount for payments
+      - name: bet06
+        description: Wage type amount for payments
+      - name: bet07
+        description: Wage type amount for payments
+      - name: bet08
+        description: Wage type amount for payments
+      - name: bet09
+        description: Wage type amount for payments
+      - name: bet10
+        description: Wage type amount for payments
+      - name: bet11
+        description: Wage type amount for payments
+      - name: bet12
+        description: Wage type amount for payments
+      - name: bet13
+        description: Wage type amount for payments
+      - name: bet14
+        description: Wage type amount for payments
+      - name: bet15
+        description: Wage type amount for payments
+      - name: bet16
+        description: Wage type amount for payments
+      - name: bet17
+        description: Wage type amount for payments
+      - name: bet18
+        description: Wage type amount for payments
+      - name: bet19
+        description: Wage type amount for payments
+      - name: bet20
+        description: Wage type amount for payments
+      - name: bet21
+        description: Wage type amount for payments
+      - name: bet22
+        description: Wage type amount for payments
+      - name: bet23
+        description: Wage type amount for payments
+      - name: bet24
+        description: Wage type amount for payments 
+      - name: bet25
+        description: Wage type amount for payments
+      - name: bet26
+        description: Wage type amount for payments
+      - name: bet27
+        description: Wage type amount for payments
+      - name: bet28
+        description: Wage type amount for payments
+      - name: bet29
+        description: Wage type amount for payments
+      - name: bet30
+        description: Wage type amount for payments
+      - name: bet31
+        description: Wage type amount for payments
+      - name: bet32
+        description: Wage type amount for payments
+      - name: bet33
+        description: Wage type amount for payments
+      - name: bet34
+        description: Wage type amount for payments
+      - name: bet35
+        description: Wage type amount for payments
+      - name: bet36
+        description: Wage type amount for payments
+      - name: bet37
+        description: Wage type amount for payments
+      - name: bet38
+        description: Wage type amount for payments
+      - name: bet39
+        description: Wage type amount for payments
+      - name: bet40
+        description: Wage type amount for payments
```

Comment:
Are these all intended to be the same description?

Reply:
Yes, found a slightly more detailed description via Google but they are all the same <img width="695" alt="Screenshot 2023-07-06 at 4 20 19 AM" src="https://github.com/fivetran/dbt_sap_source/assets/108772760/3f901c8e-a692-42d4-bd80-e14662e639f5">